client/wallet/webconnector.go

```diff
@@ -40,6 +42,12 @@ func (webConnector *WebConnector) ServerStatus() (status ServerStatus, err error
 
 // RequestFaucetFunds request some funds from the faucet for test purposes.
 func (webConnector *WebConnector) RequestFaucetFunds(addr address.Address, powTarget int) (err error) {
+	rateSetterInfo, err := webConnector.client.RateSetter()
+	if err != nil {
+		return err
+	}
+	time.Sleep(rateSetterInfo.Estimate)
+
```

Comment:
Couldn't the client expose such a functionality? And you just call it from here?